content/spin/v2/manifest-reference.md

```diff
@@ -49,6 +50,20 @@ command = "cargo build --target wasm32-wasi --release"
 watch = ["src/**/*.rs", "Cargo.toml"]
 ```
 
+## Using Variables in the Manifest
+
+The following fields allow you to use [expressions](./variables.md#adding-variables-to-your-applications) in their values:
+
+* `application.trigger.redis.address`
```

Comment:
Wondering if we also want to cross-reference this info in applicable spots elsewhere in the docs?  eg
- https://developer.fermyon.com/spin/v2/redis-trigger#redis-trigger-application-settings for `application.trigger.redis.address`
- https://developer.fermyon.com/spin/v2/redis-trigger#specifying-a-redis-trigger for `trigger.redis.channel`
- https://developer.fermyon.com/spin/v2/http-outbound#granting-http-permissions-to-components for `allowed_outbound_hosts`.

Maybe a brief note around how variables can be used in these scenarios?

Reply:
@vdice Yes, that's a great idea!  Thanks - will do!